tests/drivers/i2c/i2c_ram/testcase.yaml

```diff
@@ -3,13 +3,27 @@ common:
   tags:
     - drivers
     - i2c
```

Comment:
as arduino header is required by default, so need add 
```
depends_on: 
    - arduino_i2c
```

Reply:
perhaps I simply need to assign this test a fixture or something? It doesn't truly depend on arduino_i2c it depends on i2c-ram existing which I suppose is a test fixture of sorts?

Reply:
yes, you can do that way, and to make the cse more adaptive to new platforms, we can reuse the arduino_i2c as a sub dependency, so that all boards with arduino_i2c will enabled by default with the test fixture installed. and this can also applys to other common header definition, one step further can save a lot.

Reply:
added a fixture, i2c_ram